test/commands.test.ts

```diff
@@ -197,7 +197,7 @@ describe("install", () => {
 
       assert.match(
         pkgJson.dependencies["@std/encoding"],
-        /^npm:@jsr\/std__encoding@\d+\.\d+\.\d+.*$/,
+        /^npm:@jsr\/std__encoding@[^]?\d+\.\d+\.\d+.*$/,
```

Comment:
pnpm changed their output again. For us it makes no difference if the `^` is there, because the lockfile will have an exact version anyways.